cmake/check_symbol_conflict.py

```diff
@@ -0,0 +1,35 @@
+import platform
```

Comment:
This is totally nitpicking (not critical): I think it can be solved relatively simply with awk/sort/comm. Again, it is really not critical at all, rather just a thought. Chances are, Python version is more readable than that.